bin/proof-generator/include/nil/proof-generator/detail/utils.hpp

```diff
@@ -36,40 +36,6 @@ namespace nil {
     namespace proof_generator {
         namespace detail {
 
-            template<typename TIter>
-            void print_hex_byteblob(std::ostream& os, TIter iter_begin, TIter iter_end, bool endl) {
-                os << "0x" << std::hex;
-                for (TIter it = iter_begin; it != iter_end; it++) {
-                    os << std::setfill('0') << std::setw(2) << std::right << int(*it);
-                }
-                os << std::dec;
-                if (endl) {
-                    os << std::endl;
-                }
-            }
-
-            template<typename Endianness, typename Proof, typename CommitmentParamsType>
-            void proof_print(
-                const Proof& proof,
-                const CommitmentParamsType& params,
-                boost::filesystem::path output_file
-            ) {
-                using namespace nil::crypto3::marshalling;
-
-                using TTypeBase = nil::marshalling::field_type<Endianness>;
-
-                auto filled_placeholder_proof =
-                    crypto3::marshalling::types::fill_placeholder_proof<Endianness, Proof>(proof, params);
-
-                std::vector<std::uint8_t> cv;
-                cv.resize(filled_placeholder_proof.length(), 0x00);
-                auto write_iter = cv.begin();
-                nil::marshalling::status_type status = filled_placeholder_proof.write(write_iter, cv.size());
-                std::ofstream out;
-                out.open(output_file.c_str());
-                print_hex_byteblob(out, cv.cbegin(), cv.cend(), false);
-            }
-
             template<typename T, std::size_t Size>
             constexpr std::size_t find_index(const T& item, const std::array<T, Size>& arr) {
```

Comment:
Maybe delete the whole file? This function find_index is all that is left, and it can be replaced by std::find on the caller side.

Reply:
Thanks, removed it